docs/configuration/CONFIGURATION_SUPPORTED_EXTRACTOR.md

```diff
@@ -1,18 +1,23 @@
 # Configuration
```

Comment:
This file is missing documentation regarding the `O3rConfig` decorator

Reply:
https://github.com/AmadeusITGroup/otter/pull/1672/files#diff-405f767e79930a31ed8224976a6e4a8d47b9ba02aa43ec17734235e404277f0dR412